### ISSUE TITLE
Pi3-X evaluation

### ISSUE THREAD

**Issue by bowieshi** [yyfz/Pi3#113]
Hi! Thanks for the great work! May I ask did you evaluate Pi3-X performance? How does it compare to monocular metric depth estimator MoGe2 and multi-view metric estimator Mapanything?

**Comment by bowieshi** [comment]
Also, may I ask is there any plan to release the training code for Pi3-X? The open-source for training code surely helps a lot, thanks!

**Comment by yyfz** [comment]
> I would also be curious to know what you would expect the performance to be compared to HunyuanWorld-Mirror

Hi, we haven't compared it directly, but we expect to achieve better performance since we have more training data.

**Comment by yyfz** [comment]
Following link has some evalution of Pi3X: https://x.com/Nik__V__/status/2016556659145855417/photo/1
